src/lib/callable.ts

```diff
@@ -1,3 +1,5 @@
-const callable = val => (typeof val !== 'function' ? () => val : val)
+const callable = (possibleFunction: any) => (...args: any[]) => {
+  return typeof possibleFunction === 'function' ? possibleFunction(...args) : possibleFunction
```

Comment:
👍

Reply:
Can we attach a benchmark into JSDoc?
https://jsperf.com/startdust-callable

I thought that `callableRest()` would be faster :smirk_cat: But, not